app/assets/stylesheets/application.scss

```diff
@@ -213,6 +213,10 @@ body {
   transition: all 0.2s;
 }
 
+.hide-dom {
+  display: none;
+}
```

Comment:
I am using this CSS class to hide the index partial that the user has not selected.

Reply:
Nice!